lib/rubocop/git/commit_file.rb

```diff
@@ -28,8 +28,8 @@ def removed?
     @file.status == 'removed'
   end
 
-  def ruby?
-    filename.match(/.*\.rb$/)
+  def supported?
+    filename.match(/.*\.(rb|rake)|Gemfile|Rakefile$/)
```

Comment:
They are still ruby files, just not with the `.rb` extension 😅

Reply:
@gsamokovarov I can keep the method name, does it make more sense to be `ruby?` rather than `supported?` as you can add other files here?

```ruby
file = CommitFile.new
file.supported?

or 

file.ruby?
```